### PR TITLE
Count filtered data and update query

### PR DESCRIPTION
**Pull request by remo87**
add count for filtered data QLITAGG
add count to schema and backend

### CODE REVIEW

File: app/models/Backend.scala [hunk 1]
```diff
@@ -786,6 +788,8 @@ class Backend @Inject() (implicit
     val indexTable = defaultOTSettings.clickhouse.literatureIndex
     logger.info(s"query literature ocurrences in table ${table.name}")
 
+    println("cursor" + cursor)
```
Comment (mbdebian) [inline app/models/Backend.scala]: We should probably remove this debug message

File: app/models/Backend.scala [hunk 2]
```diff
@@ -794,6 +798,9 @@ class Backend @Inject() (implicit
       case _ => Option.empty
     }
 
+    println("pag.size" + pag.size)
```
Comment (mbdebian) [inline app/models/Backend.scala]: Let's remove this

File: app/models/Backend.scala [hunk 3]
```diff
@@ -794,6 +798,9 @@ class Backend @Inject() (implicit
...
+    println("pag.offset" + pag.offset)
```
Comment (mbdebian) [inline app/models/Backend.scala]: And this

**Review by mbdebian — CHANGES_REQUESTED** [review]
@remo87 , please, have a look at the comments on some of the files.
Thanks!

File: app/models/Backend.scala (hunk 3 above)
Comment (remo87) [reply app/models/Backend.scala]: applied

File: app/models/Backend.scala (hunk 1 above)
Comment (remo87) [reply app/models/Backend.scala]: applied

**Review by mbdebian — DISMISSED** [review]

**Pull request merged**